### ISSUE TITLE
Java 9 modules : split package org.springframework.boot.context.properties

### ISSUE THREAD

**Issue by pull-vert** [spring-projects-experimental/spring-fu#256]
This leads to this error when module-info.java requires `spring.boot` and `spring.fu.kofu` (that includes module `spring.fu.autoconfigure.adapter`) :
`java.lang.module.ResolutionException: Modules spring.boot and spring.fu.autoconfigure.adapter export package org.springframework.boot.context.properties`

**Comment by sdeleuze** [comment]
Yeah I have to do that in order to be able to access package private Spring Boo classes, but let's put that in 0.4 bucket in order to work with Spring Boot team to avoid that.
